orientation_readme.md

```diff
@@ -0,0 +1,39 @@
+# Orientation : Welcome to your first day!
+This document was made to help newcomers set up all the software that they will need to actively learn and participate in the JustAskDavidB Coaching Program.
+
+##  Downloading and Setting Up Software
+
+### [Jetbrain](https://account.jetbrains.com/login)
+We use Jetbrain products for web development.  This includes PHPStorm for PHP development, DataGrip for SQL and database interaction, and Pycharm for Python development. Downloading the trials is fine because a JetBrain license will be purchased for you as part of the JustAskDavidB Coaching Program. You will receive an email from JetBrains to complete your registration within the first 15-days. After logging into JetBrains, “Authorization Successful” should appear on screen.
```

Comment:
It's "JetBrains" with an S. Correct all instances without the S (3)
Spell out fifteen and delete the following hyphen